packages/xo-web/src/xo-app/self/index.js

```diff
@@ -569,6 +569,9 @@ export class Edit extends Component {
             >
               {_('saveResourceSet')}
             </ActionButton>
+            <ActionButton icon='cancel' handler={onCancel}>
```

Comment:
Add `onCancel` props when creating a new resourceSet because it will be undefined.